src/nrfx/hal/nrf_soc_if.h

```diff
@@ -72,6 +75,13 @@ extern NRF_TIMER_Type NRF_TIMER_regs[];
 #define NRF_TIMER4_BASE                   (&NRF_TIMER_regs[4])
 extern NRF_POWER_Type NRF_POWER_regs;
 #define NRF_POWER_BASE                    (&NRF_POWER_regs)
+extern NRF_GPIO_Type NRF_P0_regs;
+#undef NRF_P0_BASE
+#define NRF_P0_BASE                       (&NRF_P0_regs)
```

Comment:
What about P1? Shouldn't a stub for it be added as well?

Reply:
I left it out as it models the nRF52832 which does not have P1